### PR TITLE
Permission "Can use OnlyOffice"

### PR DESCRIPTION
**Pull request by yurabakhtin**
Issue: https://github.com/humhub/team_tasks/issues/125

### CODE REVIEW

File: Events.php [hunk 1]
```diff
@@ -19,6 +20,10 @@ class Events
 
     public static function onFileHandlerCollection($event)
     {
+        if (!Yii::$app->user->can(CanUseOnlyOffice::class)) {
+            //return;
```
Comment (luke-) [inline Events.php:24]: This probably shouldn't be commented out, should it?

**Review by luke- — CHANGES_REQUESTED** [review]

File: Events.php (hunk 1 above)
Comment (yurabakhtin) [reply Events.php]: Oh yes, sorry, fixed in https://github.com/ONLYOFFICE/onlyoffice-humhub/pull/42/commits/05f5e89f4fc7fceceb5a09ca72eb9e3024b51c6e

**Review by luke- — APPROVED** [review]

**Comment by LinneyS** [comment]
Hi @yurabakhtin 

https://github.com/humhub/team_tasks is private repo. Can you write a description of the issue here?

**Comment by luke-** [comment]
@LinneyS Sorry about that!

The feature allows HumHub administrators to define which user groups are allowed to access the configured OnlyOffice server.
For example, external users can be denied access and can only download documents via HumHub.

By default, all users are allowed to use the configured OnlyOffice server.

@yurabakhtin Can you please confirm, that the default of this permission is allow.

**Comment by yurabakhtin** [comment]
@luke- 

> Can you please confirm, that the default of this permission is allow.

No, by default the permission is "Deny", should I change to "Allow"?

**Comment by luke-** [comment]
@yurabakhtin Yes, please switch to "Allow". It should all work as before and only be an additional option.

**Comment by yurabakhtin** [comment]
@luke- Ok, commit https://github.com/ONLYOFFICE/onlyoffice-humhub/pull/42/commits/0032adcdbd73821cb73a35279d8f1f90293d6968.

**Review by luke- — APPROVED** [review]

**Comment by luke-** [comment]
@LinneyS Is there anything open or else against merging the PR?

**Comment by Antipkin-A** [comment]
Hi @luke-, @yurabakhtin 
We want to try this feature in the web interface, but can't find settings for allow/deny to use onlyoffice. Where can i find them?
I tried humhub-1.9.0-beta1 and humhub-1.8.0 versions

**Comment by luke-** [comment]
@Antipkin-A This permission can be controlled by user groups. `Administration -> Users -> Groups -> Permissions`.
To test a non-oo user, create a group without the OnlyOffice Permission, and add a user only to this group.

Maybe you need to flush the cache before testing (administration -> settings -> advanced -> cache -> flush).

**Comment by LinneyS** [comment]
@yurabakhtin @luke- thanks
sorry for the late reply

**Pull request merged**